packages/zoe/src/contracts/attestation/expiring/expiringNFT.js

```diff
@@ -0,0 +1,143 @@
+// @ts-check
+
+import { makeStore } from '@agoric/store';
+import { AmountMath, AssetKind } from '@agoric/ertp';
+
+// eslint-disable-next-line import/no-extraneous-dependencies
```

Comment:
Yes, it's a self-referential import

Reply:
This response confuses me. I think the comment on line 6 suppresses potential complaints about line 7. But line 7 appears to be relied on by line 62. Does line 7 require the `disable` on line 6?

Reply:
Yep, this is what it looks like otherwise.
![Screen Shot 2021-07-23 at 7 53 51 PM](https://user-images.githubusercontent.com/2441069/126855494-63df6f9b-b6b6-4398-aa15-3f7c3fcd4f91.png)

Reply:
The complaint is that `'@agoric/zoe`` isn't in the dependency list, which is odd inside the zoe package. Does it want internal dependencies to use `'../../../makeHandle'` instead?

It doesn't seem to be saying that `makeHandle` is unused, which is what I usually expect a disable for ` import/no-extraneous-dependencies` to indicate.